docs/CHANGELOG/changelog-0.1.x.md

```diff
@@ -0,0 +1,20 @@
+# v0.1.0
+
+First official release of Claudie
+
+# Features
+
+- Multi-cloud kubernetes cluster management
+- Multi-cloud loadbalancer management
+- Fast scale-up/scale-down of defined infrastructure
+- Persistent storage via Longhorn
+- Support for AWS, Azure, GCP, OCI and Hetzner
+- Support for Cloud DNS on GCP only (for now)
+
+# Bugfixes
+- As this is first release there are no bugfixes
+
+# Known issues
+- `Terraformer` sometimes timeouts when provisioning `Azure` VMs #386
+- `iptables` reset after reboot and block all traffic on `OCI` node #466
+- Numerous issues when deploying on `arm64` based machines 
```

Comment:
the note about ARM should be more explicit IMO, I think the support is no quite there yet. perhaps it could read that the only fully supported arch ATM is x86_64.

Reply:
I see your point, however, the section is `Known issues` and I do not think it should be saying what you suggested since that is not an issue. I prefer to keep it like this, I believe that is a good enough warning for people who would bother to read the changelog :).

Reply:
I think this suffices for now.
Let's review whether there are any more arm64 issues after https://github.com/Berops/claudie/issues/468 is done

Reply:
> I see your point, however, the section is `Known issues` and I do not think it should be saying what you suggested since that is not an issue. I prefer to keep it like this, I believe that is a good enough warning for people who would bother to read the changelog :).

hm, alright.

probably anybody we think is interested in releases would read the changelog.

the note doesn't have to be in the known issues section, it could be elsewhere, what I pointed at was the visibility..